services/ui-src/src/forms/adminDashSelector/adminDashSelector.ts

```diff
@@ -33,7 +33,7 @@ export default {
       type: "dropdown",
       validation: "dropdown",
       props: {
-        hint: "Select state or territory:",
+        label: "Select state or territory:",
```

Comment:
CMSDS requires a label for all its dropdowns and will throw an absolute fit in the console if you do not provide one.